Engine/ac/string.cpp

```diff
@@ -257,6 +258,52 @@ const char* String_UpperCase(const char *thisString) {
     return CreateNewScriptString(std::move(buf));
 }
 
+void * String_Split(const char *thisString, const char *separator)
+{
+    if ((thisString == nullptr) || (thisString[0] == 0))
+        return nullptr;
+
+    auto const &header = ScriptString::GetHeader((void*)thisString);
+
+    if ((separator == nullptr) || (separator[0] == 0)) {
+        std::vector<const char*> items{};
+        items.push_back(thisString);
+        DynObjectRef arr = DynamicArrayHelpers::CreateStringArray(std::vector<const char*>(items));
```

Comment:
This is probably a typo, should be passing `items`, because right now you are constructing a new temporary vector.

Reply:
thanks! This was leftover from a code that did all in one line - but looked ugly...